### PR TITLE
Move restored entity warning up and use ha-alert

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Move the restore entity text up and into an ha-alert.

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-alert.ts [hunk 1]
```diff
@@ -83,7 +83,6 @@ class HaAlert extends LitElement {
       position: relative;
       padding: 8px;
       display: flex;
-      margin: 4px 0;
```
Comment (zsarnett) [inline src/components/ha-alert.ts:86]: How is this affecting other places?

File: src/translations/en.json [hunk 2]
```diff
@@ -776,11 +776,8 @@
           "activity": "Current activity"
         },
         "restored": {
-          "not_provided": "This entity is currently unavailable and is an orphan to a removed, changed or dysfunctional integration or device.",
-          "remove_intro": "If the entity is no longer in use, you can clean it up by removing it.",
-          "remove_action": "Remove entity",
-          "confirm_remove_title": "Remove entity?",
-          "confirm_remove_text": "Are you sure you want to remove this entity?"
+          "title": "Entity unavailable",
+          "not_provided": "This entity is no longer being provided by the {integration} integration. If the entity is no longer in use, delete it in settings."
```
Comment (zsarnett) [inline src/translations/en.json:780]: Should we change the key here?
```suggestion
          "no_longer_provided": "This entity is no longer being provided by the {integration} integration. If the entity is no longer in use, delete it in settings."
```

File: src/components/ha-alert.ts (hunk 1 above)
Comment (balloob) [reply src/components/ha-alert.ts:86]: I need to check that. We should never have a margin on the outside of a component. It's up to the parent how different components need to be styled together.

File: src/translations/en.json (hunk 2 above)
Comment (balloob) [reply src/translations/en.json]: done.

File: src/components/ha-alert.ts (hunk 1 above)
Comment (balloob) [reply src/components/ha-alert.ts:86]: Fixed the design docs CSS. It was rendering multiple next to one another. Shouldn't happen in real code.

**Review by zsarnett — APPROVED** [review]

**Pull request merged**